src/main/java/de/tum/cit/aet/artemis/programming/service/jenkins/JenkinsEndpoints.java

```diff
@@ -9,19 +10,36 @@
 
 public enum JenkinsEndpoints {
 
-    NEW_PLAN("job", "<projectKey>", "createItem"), NEW_FOLDER("createItem"), DELETE_FOLDER("job", "<projectKey>", "doDelete"),
-    DELETE_JOB("job", "<projectKey>", "job", "<planName>", "doDelete"), PLAN_CONFIG("job", "<projectKey>", "job", "<planKey>", "config.xml"),
-    TRIGGER_BUILD("job", "<projectKey>", "job", "<planKey>", "build"), ENABLE("job", "<projectKey>", "job", "<planKey>", "enable"),
-    TEST_RESULTS("job", "<projectKey>", "job", "<planKey>", "lastBuild", "testResults", "api", "json"),
-    LAST_BUILD("job", "<projectKey>", "job", "<planKey>", "lastBuild", "api", "json");
+    // @formatter:off
+    // Build plan endpoints
+    NEW_PLAN("job", "<projectKey>", "createItem"),
+    NEW_FOLDER("createItem"),
+    DELETE_FOLDER("job", "<projectKey>", "doDelete"),
+    DELETE_JOB("job", "<projectKey>", "job", "<planName>", "doDelete"),
+    PLAN_CONFIG("job", "<projectKey>", "job", "<planKey>", "config.xml"),
+    FOLDER_CONFIG("job", "<projectKey>", "config.xml"),
+    TRIGGER_BUILD("job", "<projectKey>", "job", "<planKey>", "build"),
+    ENABLE("job", "<projectKey>", "job", "<planKey>", "enable"),
+    LAST_BUILD("job", "<projectKey>", "job", "<planKey>", "lastBuild", "api", "json"),
+    GET_FOLDER_JOB("job", "<projectKey>", "api", "json"),
+    GET_JOB("job", "<projectKey>", "job", "<planKey>", "api", "json"),
+
+    // Health
+    HEALTH("login"),
+
+    // User management endpoints
+    GET_USER("user", "<username>", "api", "json"),
+    DELETE_USER("user", "<username>", "doDelete"),
+    CREATE_ADMIN("securityRealm", "createAccountByAdmin");
```

Comment:
```suggestion
    CREATE_USER("securityRealm", "createAccountByAdmin");
```
This creates any new user via the admin API rather than creating a new admin user in Jenkins.